TEveEventDisplay/src/TEveMu2eDataInterface.cc

```diff
@@ -290,6 +290,104 @@ namespace mu2e{
   }
 
 
+using LHPT = KinKal::PiecewiseTrajectory<KinKal::LoopHelix>;
+using CHPT = KinKal::PiecewiseTrajectory<KinKal::CentralHelix>;
+using KLPT = KinKal::PiecewiseTrajectory<KinKal::KinematicLine>;
+template<class KTRAJ> void TEveMu2eDataInterface::AddKinKalTrajectory( std::unique_ptr<KTRAJ> &trajectory, TEveMu2eCustomHelix *line, TEveMu2eCustomHelix *line_twoDXY, TEveMu2eCustomHelix *line_twoDXZ){
+  double t1=trajectory->range().begin();
+  double t2=trajectory->range().end();
+
+  double x1=trajectory->position3(t1).x();
+  double y1=trajectory->position3(t1).y();
+  double z1=trajectory->position3(t1).z();
+  GeomHandle<DetectorSystem> det;
+  XYZVectorF pos(x1,y1,z1);
+  CLHEP::Hep3Vector p = GenVector::Hep3Vec(pos);
+  CLHEP::Hep3Vector InMu2e = det->toMu2e(p);
+  line->SetPoint(0,pointmmTocm(InMu2e.x()), pointmmTocm(InMu2e.y()) , pointmmTocm(InMu2e.z()));
+  line_twoDXY->SetPoint(0,pointmmTocm(x1), pointmmTocm(y1) , pointmmTocm(z1));
+  line_twoDXZ->SetPoint(0,pointmmTocm(x1), pointmmTocm(y1) , pointmmTocm(z1));
+  for(double t=t1; t<=t2; t+=0.1)
```

Comment:
Can the time step be configurable? 100ps is ~30mm, which is quite coarse

Reply:
I could change by a factor of 10 i.e. 0.01 (or any toher number you want)
I dont think havig it as a FCL parameter is a good idea. We already have such a large amount of parameters